lib/ansible/module_utils/redfish_utils.py

```diff
@@ -378,12 +392,12 @@ def restart_manager_gracefully(self):
             return response
         return {'ret': True}
 
-    def manage_system_power(self, command):
+    def manage_system_power(self, command, systems_uri):
```

Comment:
For manage_system_power() and the other methods called from redfish_config.py and redfish_command.py, the `systems_uri` parameter is not being supplied by the caller. A couple of possible ways to resolve this:

1. Update redfish_config.py and redfish_command.py to call the multi (or aggregate) version of the methods.
2. Update manage_system_power() (and the other methods called from redfish_config.py and redfish_command.py) to get the first system URI from the systems_uris array and act on that. This would keep the existing behavior for the redfish_config.py and redfish_command.py modules, but those could then be updated by a subsequent issue and PR.

The downside to (1) is that when doing an update operation (create/delete/modify) it will by default do so for every system in the service. That is a valid user scenario, but for update operations it is is just as likely (if not more likely) that it should only apply to a particular system.

@tadeboro What is your opinion on dealing with the update operations in this PR?

Reply:
@billdodd @tadeboro thanks for the review.  I followed the second way. Let me know what you think :)